AUTHORS

```diff
@@ -4,3 +4,19 @@
 # Initial version authors:
 ChaiShushan <chaishushan@gmail.com>
 
+# Forked version authors:
+Chih-Wei Chang <>
+MohamedHelala <mohamed.helala@gmail.com>
+Adrian Zankich <>
+vanillahsu <>
+Alexander Kirillov <>
+Aditya Bhargava <>
+Bradley Griffiths <>
+Ashkan Kiani <>
+Peter Bui <>
+satojkovic <>
+TEE JIA HEN <>
+chai2010 <>
+
+
+
```

Comment:
Yes, I did test them and everything works fine. These modifications were actually part of a larger project but I decided to share them. I just want you to look at the modifications I did to Iplimage and Mat. It is hard to repeat the implementation of functions such as Set2D, Get2D with every type. It will be hard to maintain this in the future because opencv has many functions that takes CvArr. I created a Ptr() method which returns unsafe.Pointer and can be used with function calls. I also suggest to have the Ptr() method as an interface.